lib/rake/circle.rake

```diff
@@ -195,21 +195,15 @@ def test_eyes?
 end
 
 def start_sauce_connect
-  # Use latest sauce connect client for each run so we don't have to keep up with updates and end-of-lifes.
-  # If a newly-released version breaks the build, a quick fix to unblock the issue is to temporarily
-  # pin the version we use to the last working version, while we schedule the task to get the upgraded version
-  # working. You can do this by replacing `sc_download_url` with a hard-coded download url.
-
-  sc_version_info = JSON.parse(Net::HTTP.get(URI('https://saucelabs.com/versions.json')))
-  sc_download_url = sc_version_info['Sauce Connect']['linux']['download_url']
+  sc_download_url = 'https://saucelabs.com/downloads/sauce-connect/5.2.1/sauce-connect-5.2.1_linux.x86_64.tar.gz'
   tar_name = sc_download_url.split('/')[-1]
   dir_name = tar_name.chomp('.tar.gz')
 
   RakeUtils.system_stream_output "wget --quiet #{sc_download_url}"
   RakeUtils.system_stream_output "tar -xzf #{tar_name}"
   Dir.chdir(Dir.glob(dir_name)[0]) do
     # Run sauce connect a second time on failure, known periodic "Error bringing up tunnel VM." disconnection-after-connect issue, e.g. https://circleci.com/gh/code-dot-org/code-dot-org/20930
-    RakeUtils.exec_in_background "for i in 1 2; do ./bin/sc -P 4445 -l $CIRCLE_ARTIFACTS/sc.log -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -i #{CDO.circle_run_identifier} --tunnel-domains *.code.org,*.csedweek.org,*.hourofcode.com,*.codeprojects.org && break; done"
+    RakeUtils.exec_in_background "for i in 1 2; do ./bin/sc -P 4445 -l $CIRCLE_ARTIFACTS/sc.log -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -i #{CDO.circle_run_identifier} --tunnel-domains .*\\.code.org,.*\\.csedweek.org,.*\\.hourofcode.com,.*\\.codeprojects.org --tunnel-name cdo-tunnel && break; done"
```

Comment:
1. sc v5 requires `--tunnel-name` to be explicitly set
2. sc v5 requires `--tunel-domains` to be in regex format (rather than shell glob): UGH, but fine whatever